src/selectors/getApiRequest.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getApiDataRequest } from './getApiDataRequest';
+import { getState } from '../mocks/mockActions';
+
+test('it gets the request status of a given endpoint', () => {
+    const testOneEndpoint = 'data';
+    const testTwoEndpoint = 'oberon';
+    const params = { one: 'one', two: 'two' };
+
+    expect(getApiDataRequest(getState('data', true, {}, 'ready'), testOneEndpoint)).toEqual(
+        {
+            lastCall: 10,
+            duration: 0,
+            networkStatus: 'ready'
+        }
+        );
+    expect(getApiDataRequest(getState('oberon', true, { oberon: params }, 'ready'), testTwoEndpoint, params)).not.toEqual(
```

Comment:
Pls remove the "not" and test against what you do expect. There's an infinite number of possible values that the return value will not equal, but only one that will match (deterministic function). So it makes sense to test against the expected value.